src/nuget/Fable.JsonConverter/Fable.JsonConverter.fs

```diff
@@ -34,10 +35,18 @@ type JsonConverter() =
                 read (index + 1) (acc @ [value])
         advance reader
         read 0 List.empty
+
+    let typeCache = Dictionary<Type, bool>()
```

Comment:
I think this is very sensible. As we don't have control over when `CanConvert` will be called, shall we make this a `ConcurrentDictionary` to be on the safe side?